metrics-operator/chart/values.yaml

```
@@ -93,6 +93,8 @@ annotations: {}
 podAnnotations: {}
 ## @param     kubernetesClusterDomain overrides cluster.local
 kubernetesClusterDomain: cluster.local
+## @param   enableCustomMetricsAPIService enable/disable the K8s APIService installation
+enableCustomMetricsAPIService: "true"
```

i would prefer to use something like
```suggestion
customMetricsAPIService:
  enabled: "true"
```
This gives more flexibility in the future to extend this part of the helm values with more options